test/bundles/bundles.test.ts

```diff
@@ -40,7 +40,13 @@ describe('bundles', () => {
 
   it('IIFE bundle - should generate BPMN Diagram SVG', async () => {
     const pageTester = new BpmnStaticPageSvgTester(
-      { pageFileName: 'lib-integration-iife', expectedPageTitle: 'BPMN Visualization IIFE bundle', bpmnContainerId: 'bpmn-container-for-iife-bundle' },
+      {
+        targetedPage: {
+          pageFileName: 'lib-integration-iife',
```

Comment:
It is possible but I decided to not add this page on purpose.

In my initial implementation, I put it there. But then, I realized that it is not a page which is available through the development server as other pages in `AvailableTestPages`. It is a special page we access via local file browsing for the sole purpose of testing the IIFE integration. There is also special code in the `PageTester` to access the page.
So to mark it as special, and to avoid mistakes (i.e. try using the page in regular tests), I didn't put with other pages reference.

If we have other pages for iife integration in the future, we could introduce the same mechanism dedicated for such pages.